components/cam/src/physics/crm/crm_history.F90

```diff
@@ -488,73 +487,73 @@ subroutine crm_history_out(state, ptend, crm_state, crm_rad, crm_output, crm_ecp
    call outfld('CRM_T   ',crm_state%temperature, pcols, lchnk )
 
    if (MMF_microphysics_scheme .eq. 'sam1mom') then
-      call outfld('CRM_QV  ',(crm_state%qt-crm_output%qcl-crm_output%qci), pcols, lchnk )
+      call outfld('CRM_QV  ',(crm_state%qt-crm_output_qcl-crm_output_qci), pcols, lchnk )
```

Comment:
@whannah1 , @brhillman How do yall feel about these changes: removing the `crm_output` data structure and putting flat arrays in their place? I think I would rather see the data structure remain, and instead we just change the call to `crm(...)` to pass these arrays individually.

Reply:
Yeah, I agree Matt.